### PR TITLE
CS01 Interpolation

### PR DESCRIPTION
**Pull request by Sameem25**
The following program can predict future profits for the corresponding year upon the request of the user.

### CODE REVIEW

File: pset0/sameem25/CS01.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+#user input values for 'Year' and the corresponding 'Profit'
+year_arr = []
+profit_arr = []
+user = (input('Enter the number of data to be presented: '))
+for i in range(int(user)):
+    year = float(input('Enter the year: '))
+    year_arr.append(year)
+    profit= float(input("Enter the profit for the year: "))
```
Comment (RudraNilBasu) [inline pset0/sameem25/CS01.py]: A space before the = also (Same for lines 25 and 26)

File: pset0/sameem25/CS01.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+#converting them to numpy arrays as column matrix
+year= np.array(year_arr).reshape(int(user),1)
+profit= np.array(profit_arr).reshape(int(user),1)
+
+
+
+#This part has been designed to work on train-set using ML algorithm.
+length= len(year)
+for i in range(length):
+    b=np.insert(year, 0, 1, axis=1)
```
Comment (RudraNilBasu) [inline pset0/sameem25/CS01.py]: Space before and after =

File: pset0/sameem25/CS01.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+c=b[:,1]**2
```
Comment (RudraNilBasu) [inline pset0/sameem25/CS01.py]: Here too.

**Review by RudraNilBasu — CHANGES_REQUESTED** [review]
Thanks for the PR, can you squash the commits in a single commit following the format: https://bytesclub.github.io/contributing/#tips-for-contribution

**Comment by Sameem25** [comment]
The changes have been made.

**Review by RudraNilBasu — COMMENTED** [review]
Regarding the commit messages, can you break them at 80 character limits as present in BytesClub.github.io/contributing ?

**Comment by Sameem25** [comment]
Well, now the commit message is according to Bytes Club's rule.

**Review by RudraNilBasu — APPROVED** [review]

**Review by 0xTheProDev — CHANGES_REQUESTED** [review]
Rename the file to be _**Interpolation.py**_ and remove additional print statements. Otherwise good to go!

**Comment by Sameem25** [comment]
Well, the file name is changed and the print statements as well. Note there is only one print statement at the end to show output and no additional 'print' statements.

**Pull request merged**